package.json

```diff
@@ -48,8 +43,8 @@
     "cross-env": "^7.0.3",
     "is-canonical-base64": "^1.1.1",
     "scuttle-testbot": "^2.1.0",
-    "ssb-backlinks": "^2.1.1",
-    "ssb-query": "^2.4.5",
+    "ssb-box2": "ssbc/ssb-box2#pobox",
+    "ssb-db2": "^7.1.1",
     "ssb-replicate": "^1.3.3",
     "standard": "^17.1.0",
     "tap-arc": "^0.4.0",
```

Comment:
TODO: consider write `peerDependencies` for the required modules... another PR tho